HotFix.Demo.Acceptor/Program.cs

```diff
@@ -33,6 +38,17 @@ static void Main(string[] args)
                 OutboundSeqNum = 1,
                 HeartbeatInterval = 0,
                 //LogFile = @"messages.log" // Enable to see logging impact
+                Schedules = new List<ISchedule>
+                {
+                    new Schedule
+                    {
+                        Name = "Session",
+                        OpenDay = day,
+                        OpenTime = time,
+                        CloseDay = day,
+                        CloseTime = time - TimeSpan.FromSeconds(1)
```

Comment:
I don't think we need anything elaborate here for the scheduling, something like this would be a lot clearer:

```csharp
Schedules = new List<ISchedule>
{
    new Schedule
    {
        Name = "Session",
        OpenDay = DayOfWeek.Monday,
        OpenTime = TimeSpan.Parse("00:00:00"),
        CloseDay = DayOfWeek.Sunday,
        CloseTime = TimeSpan.Parse("23:59:59")
    }
```